web/client/translations/data.hr-HR

```diff
@@ -1888,6 +1888,39 @@
                 "fit": "Fit the container space",
                 "editMedia": "Change media source"
             },
+            "builder": {
```

Comment:
Last time we discussed it we decided that populating with english these files was a bad practice
Did something changed I don't know?

Reply:
ok, sorry, i'll change this in the future..